src/pybind/mgr/cephadm/utils.py

```diff
@@ -43,6 +43,13 @@ class ContainerInspectInfo(NamedTuple):
     repo_digests: Optional[List[str]]
 
 
+class SpecialHostLabels():
+    admin_label: str = '_admin'
+    no_autotune_memory_label: str = '_no_autotune_memory'
+    drain_daemons_label: str = '_no_schedule'
+    drain_conf_keyring_label: str = '_no_conf_keyring'
```

Comment:
nit: it sounds like a good place to document what every label used-for/does :)

Reply:
with this PR adding docs for the `_no_conf_keyring` label, I think they should all be covered in https://docs.ceph.com/en/latest/cephadm/host-management/#special-host-labels. It's technically docs for users rather than devs, but the behavior outlined there is basically what I would say in any code comment here anyway.